chain-signatures/node/src/protocol/mod.rs

```diff
@@ -227,6 +228,9 @@ impl MpcSignProtocol {
         loop {
             let protocol_time = Instant::now();
             tracing::trace!("trying to advance chain signatures protocol");
+            // Hardware metric refresh
+            update_system_metrics(&my_account_id);
```

Comment:
How often is this executed in practice?

Reply:
@ppca might know better than I 😂

Reply:
as often as a loop. Might be too often in this case. I think we can do once every minute?
add
`let mut last_hardware_pull = Instant::now();` at around here: https://github.com/near/mpc/blob/11dd7e062b06d059fa22a4bfef3115b124435ce7/chain-signatures/node/src/protocol/mod.rs#L215

and 
```
if last_hardware_pull.elapsed() > Duration::from_secs(60) {
    update_system_metrics(&my_account_id);;
}
``` at around https://github.com/near/mpc/blob/11dd7e062b06d059fa22a4bfef3115b124435ce7/chain-signatures/node/src/protocol/mod.rs#L229 

This will update the metrics every minute. @volovyks @kmaus-near

Reply:
Can we do that every ~10 seconds? I'm afraid we can lose CPU spikes.

Reply:
yeah, let's change to `if last_hardware_pull.elapsed() > Duration::from_secs(10)` then

Reply:
I think the example the crate uses is 5s, which would be pretty common practice for hardware metrics, and is the lowest refresh rate in grafana.

Reply:
5s sounds good to me